treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -58,6 +58,7 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity):
             'platform': platform,
             'framework': framework,
             'extra_properties': extra_properties,
+            'parent_signature': parent_signature,
```

Comment:
We just omit the property in the case that a signature does not have a parent:

https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/performance_data.py#L91